src/DisplayEmailComponents/DisplayEmailContainer/DisplayEmailContainer.js

```diff
@@ -0,0 +1,22 @@
+import EmailCard from "../DisplayEmailCard/EmailCard";
+
+const DisplayEmailContainer = (props) => {
+    const dateConvert = (emailItem) => {
+        const headerDateUnix = Date.parse(emailItem.date_created)
+        const headerDate = new Date(headerDateUnix)
+        return headerDate.toLocaleDateString("en-GB")
+    }
```

Comment:
1. i think it would be better if this function takes the date only rather than the whole email item
2. i think a more semantic name for this function would be something like `formatDateForDisplay` or something else that explains what it's converting it to